config/locales/models/transaction.en.yml

```diff
@@ -19,6 +19,7 @@ en:
         providing_organisation_type: Providing organisation type
         receiving_organisation_name: Receiving organisation name
         receiving_organisation_type: Receiving organisation type
+        value: Transaction amount
```

Comment:
Where does this get used? Is the label automagically picked out, as I can't find it in any of the views?

Reply:
Also on line 12 of this file is `curreny: Currency` (misspelling of the key). I know this isn't in this PR but I'm concerned that this key is misspelled!

Reply:
Yep, automagic. We have configured the GOVUK form builder to use `form.label.activity` see:

https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/ffe685f25e24b55b8e6c07686c41a0a6571e8def/config/initializers/form_builder.rb#L10

Reply:
No  probs on the typo - I'll fix it…

Reply:
Love a bit of magic